contracts/math/GentleSafeMath.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity ^0.4.24;
+
+
+/**
+ * @title GentleSafeMath
+ * @dev Math operations with safety checks that revert on error
+ */
+library GentleSafeMath {
+
+  /**
+  * @dev Multiplies two numbers, reverts on overflow.
+  */
+  function mul(uint256 a, uint256 b) internal pure returns (uint256 c) {
+    // Gas optimization: this is cheaper than requiring 'a' not being zero, but the
+    // benefit is lost if 'b' is also tested.
+    // See: https://github.com/OpenZeppelin/openzeppelin-solidity/pull/522
+    if (a == 0) {
+      return 0;
+    }
+
+    c = a * b;
+    require(c / a == b);
+    return c;
+  }
+
+  /**
+  * @dev Integer division of two numbers, truncating the quotient.
+  */
+  function div(uint256 a, uint256 b) internal pure returns (uint256) {
+    // uint256 c = a / b;
+    // require(a == b * c + a % b); // There is no case in which this doesn't hold
+    require(b > 0); // Solidity only automatically asserts when dividing by 0
```

Comment:
This is an interesting point; the EVM will throw an invalid op code if it tries to divide by 0. To maintain consistency, I've added this to make sure only reverts are thrown.